client/src/components/Heading/stories.tsx

```diff
@@ -0,0 +1,49 @@
+import { Story, Meta } from '@storybook/react'
+import Heading, { HeadingProps } from '.'
+
+export default {
+  title: 'Heading',
+  component: Heading,
+  argTypes: {
+    children: {
+      type: 'string'
+    },
+    size: {
+      options: [
+        'xsmall',
+        'small',
+        'medium',
+        'normal',
+        'large',
+        'xlarge',
+        'huge'
+      ],
+      control: { type: 'select' },
+      defaultValue: 'normal'
+    },
```

Comment:
O Storybook já faz select por padrão nos casos de várias options, num podia passar só o default? Assim se você tiver novas variáveis, o Storybook já vai saber automaticamente.

```suggestion
    size: {
      defaultValue: 'normal'
    },
```